packages/gafl-webapp-service/src/processors/recurring-pay-helper.js

```diff
@@ -13,4 +14,5 @@ export const validForRecurringPayment = permission =>
   process.env.SHOW_RECURRING_PAYMENTS?.toLowerCase() === 'true' &&
   permission.licenceLength === '12M' &&
   permission.isLicenceForYou &&
+  permission.licensee.age > JUNIOR_MAX_AGE + 1 &&
```

Comment:
The main problem is that there is no `age` field for licensee, only a `birthDate` field so age will have to be calculated from that.

Furthermore, if age is greater than JUNIOR_MAX_AGE + 1, that'll be 17, meaning recurring payments won't be offered until 18 (as your tests show) when the angler paid for an adult licence the previous year. If you get rid of the `+1` I think this would work, so `age > JUNIOR_MAX_AGE`